src/init/init.c

```diff
@@ -44,6 +44,7 @@ int master_core_init(struct sof *sof)
 	init_heap(sof);
 
 #if CONFIG_TRACE
+	trace_point(TRACE_BOOT_SYS_TRACES);
 	trace_init(sof);
```

Comment:
Does it make sense to call trace_point before trace_init?

Reply:
It's different trace mechanism.

Reply:
ok.